.changelog/unreleased/improvements/1755-batch-save-block.md

```diff
@@ -0,0 +1,2 @@
+- `[store]` Save block using a single DB batch ([\#1755](https://github.com/cometbft/cometbft/pull/1755))
+  Only if block is less than 640kB, otherwise each block part is saved individually.
```

Comment:
```suggestion
- `[store]` Save block using a single DB batch if block is less than 640kB, otherwise each block part is saved individually
  ([\#1755](https://github.com/cometbft/cometbft/pull/1755))
```